### ISSUE TITLE
BigEarthNet Result Reproduce

### ISSUE THREAD

**Issue by Ehzoahis** [antofuller/CROMA#7]
Hi! Thank you for your amazing work! We are currently reproducing your results on BigEarthNet but are having a significant gap in the results. Therefore, we want to ask for some clarification:
1. For the mAP metric, are you using micro-mAP or macro-mAP?
2. For the reported dataset split, you are using 10% of the 354,200 training samples for training, and the entire validation set for evaluation. However, the training set for the original BigEarthNet is 269,695; the test set is 125,866 and the validation set is 123,723. Therefore, I would like to ask how is the 354,200 training samples formulated, and are you using the validation set for evaluation or the testing set?

Thank you for your attention to this question. Hope you have a nice day!
